nslsii/motor_devices/slits.py

```diff
@@ -0,0 +1,151 @@
+from ophyd import (Device, Component, EpicsMotor, PVPositioner, EpicsSignal,
+                   EpicsSignalRO)
+from ophyd.device import create_device_from_components
+
+
+def slit(name='', axes={'hg': '-Ax:HG}Mtr', 'hc': '-Ax:HC}Mtr',
```

Comment:
mutable default values are technically possible, but are very bad practice.   It is better to default to `None` and have this dictionary in the body.  That also gives a nice path to overriding just one of the names as you can do

```
dflt = {}
dflt.update(axes)
```

Reply:
do we want to make this function private?

Reading down a bit more, no-default maybe better here.

I also think that we should not have a default name and force the callers of this function to set a class name.

Reply:
I have made the default `None`, and added an `if default is None:` section to set the default.

I am against having no-default, as having a 'default' helps with the social engineering of moving everyone to a common solution, but provides the flexibility o have custom cases. !-).

I have made `name` non-optional.